test/integration/resultsPagePostcode.js

```diff
@@ -107,7 +107,7 @@ describe('The results page', () => {
         iExpect.htmlWith200Status(err, res);
         const $ = cheerio.load(res.text);
 
-        expect($('.results__header--none').text()).to.be.equal(`There are no pharmacies within 20 miles of ${outcode}`);
+        expect($('.results__header--none').text()).to.be.equal(`We can't find any pharmacies near ${outcode}`);
```

Comment:
Can this be extended to include these 2 checks:
- https://github.com/nhsuk/connecting-to-services/pull/393/commits/951b40d7f1af2bd2bf57bf0bb45deaa681667cce#diff-d47eaffdfc3f1a1634cee7b96469fa02R173
- https://github.com/nhsuk/connecting-to-services/pull/393/commits/951b40d7f1af2bd2bf57bf0bb45deaa681667cce#diff-d47eaffdfc3f1a1634cee7b96469fa02R175

This covers the additional text that should be included in this scenario.